rcl/include/rcl/logging_rosout.h

```diff
@@ -28,25 +28,6 @@ extern "C"
 {
 #endif
 
-/// The default qos profile setting for topic /rosout
-/**
- * - depth = 1000
- * - durability = RMW_QOS_POLICY_DURABILITY_TRANSIENT_LOCAL
- * - lifespan = {10, 0}
- */
-static const rmw_qos_profile_t rcl_qos_profile_rosout_default =
```

Comment:
this is the only profile managed by `rcl`. this can be moved to `rmw` with other preset profiles.